aws-android-sdk-core/src/test/java/com/amazonaws/util/json/JsonUtilsTest.java

```diff
@@ -44,6 +44,24 @@
             + "\"object\":{}"
             + "}";
 
+    private static final String JSON_STRING_WITH_ARRAYS = "{\"string\":\"string\","
```

Comment:
If we don't already have it, we should add a facility like Amplify Android's [`Resources.java`](https://github.com/aws-amplify/amplify-android/blob/main/testutils/src/main/java/com/amplifyframework/testutils/Resources.java#L33-L36). It will allow you to keep unmodified json in `src/test/resources/core/json-string-with-arrays.json`, and then read it into test:
```java
String jsonStringWithArrays = 
    Resources.readAsJson(context, "core/json-string-with-arrays.json");
```

Reply:
May I propose that we take that on as a refactoring task separate from this PR? We do not have that facility and I would need more time to look at what it looks like.

Reply:
I think that's okay, particularly given that line 36 currently uses the same technique that you are.